### PR TITLE
feat(cb2-6641, cb2-7727, cb2-7044): dbas for lgvs

### PR DESCRIPTION
**Pull request by mcadenas-bjss**
# Merge VTMDEV-2 into develop

[CB2-6641](https://dvsa.atlassian.net/browse/CB2-6641)
[CB2-7727](https://dvsa.atlassian.net/browse/CB2-7727)
[CB2-7044](https://dvsa.atlassian.net/browse/CB2-7044)

### CODE REVIEW

File: src/app/forms/templates/test-records/create-master.template.ts [hunk 1]
```diff
@@ -492,8 +492,8 @@ export const contingencyTestTemplates: Record<VehicleTypes, Partial<Record<keyof
       required: CreateRequiredSectionLgvCar,
       vehicle: ContingencyVehicleSectionDefaultPsvHgvLight,
       test: ContingencyTestSectionSpecialistGroup1,
-      defects: DefectsTpl,
-      customDefects: CustomDefectsHiddenSection,
+      defects: defectsHiddenSection,
```
Comment (gjulien-bjss) [inline src/app/forms/templates/test-records/create-master.template.ts:495]: do we not want to remove the hidden section? I thought we changed the backend so it was no longer required, and leaving it in caused some bugs for the result of test calculation?

**Review by gjulien-bjss — COMMENTED** [review]
Also merge conflicts

File: src/app/forms/templates/test-records/create-master.template.ts (hunk 1 above)
Comment (mcadenas-bjss) [reply src/app/forms/templates/test-records/create-master.template.ts]: This needed rebasing from develop.

**Review by shivangidas — DISMISSED** [review]

**Review by leifkemp-bjss — APPROVED** [review]

**Review by shivangidas — DISMISSED** [review]

**Review by andrewf-bjss — CHANGES_REQUESTED** [review]
What is this for please? No title, ticket or story?

**Review by owen-corrigan-bjss — DISMISSED** [review]

**Review by shivangidas — APPROVED** [review]

**Comment by tahayazici** [comment]
Links to associated JIRA stories have been added @andrewf-bjss.

**Review by andrewf-bjss — COMMENTED** [review]
Happy once other comments have been resolved

**Review by andrewf-bjss — APPROVED** [review]
Happy once other comments have been resolved

**Pull request merged**